spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureCloudConfigAutoConfiguration.java

```diff
@@ -5,37 +5,30 @@
  */
 package com.microsoft.azure.spring.cloud.config;
 
-import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.cloud.endpoint.RefreshEndpoint;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.scheduling.TaskScheduler;
-import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
 
 @Configuration
 @ConditionalOnProperty(prefix = AzureCloudConfigProperties.CONFIG_PREFIX, name = "enabled", matchIfMissing = true)
 public class AzureCloudConfigAutoConfiguration {
-    public static final String WATCH_TASK_SCHEDULER_NAME = "azureConfigWatchTaskScheduler";
 
     @Configuration
     @ConditionalOnClass(RefreshEndpoint.class)
     @ConditionalOnProperty(prefix = AzureCloudConfigProperties.CONFIG_PREFIX, name = "watch.enabled")
     static class CloudWatchAutoConfiguration {
         @Bean
         public AzureCloudConfigWatch getConfigWatch(ConfigServiceOperations operations,
-                                                    AzureCloudConfigProperties properties,
-                                                    @Qualifier(WATCH_TASK_SCHEDULER_NAME) TaskScheduler scheduler,
-                                                    AzureConfigPropertySourceLocator sourceLocator) {
-            return new AzureCloudConfigWatch(operations, properties, scheduler, sourceLocator.getStoreContextsMap());
+                AzureCloudConfigProperties properties,
```

Comment:
Just question, is the code format changed? There is project specific codestyle format doc [here](https://github.com/microsoft/spring-cloud-azure/tree/master/codestyle).